### PR TITLE
Ben Contributions

### PR DESCRIPTION
**Pull request by robinbenitezmora**
Here we are practicing the contributions work and collaborations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,18 +161,14 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
     <h2 class="seventh-sheet__contact-me">Contact me</h2>
     <p class="seventh-sheet__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
     <form action="/">
```
Comment (m05e5) [inline index.html]: Great job.
- [ ] please as part of the requirement, the form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

File: index.html [hunk 2]
```diff
@@ -161,18 +161,14 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
...
-        <input class="form__button-area form__button-text" type="email" id="e-mail" placeholder="user_name@mail.com">
-        </label>
-      <label for="message-area">
-          <input class="form__button-area--size form__button-text" type="text" id="message-area" placeholder="Write your message here">
-      </label>
-      <a href="#" class="seventh-sheet__button seventh-sheet__button-text">Get It Touch</a>
-      <div class="end-line"><img src="./images/end-line.png" alt="end-line"></div>
+          <input class="form__button-area form__button-text" type="text" id="name" placeholder="Your name" maxlength="30" required>
+          <input class="form__button-area form__button-text" type="email" id="e-mail" placeholder="user_name@mail.com" required>
+          <input class="form__button-area--size form__button-text" type="text" id="message-area" placeholder="Write your message here" maxlength="500" required>
```
Comment (m05e5) [inline index.html]: - [ ] I recommend changing the message `input` tag to `textarea`. It is much more appropriate for long text inputs

File: index.html [hunk 3]
```diff
@@ -161,18 +161,14 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
...
+  
+          <input type="button" class="seventh-sheet__button" value="Get It Touch">
```
Comment (m05e5) [inline index.html]: - [ ] As we are dealing with a form, I recommend changing the input type from `button` to `submit`

File: style.css [hunk 4]
```diff
@@ -651,19 +651,9 @@ span {
...
-  line-height: 24px;
-  align-items: center;
-  letter-spacing: 0.03em;
-  text-decoration: none;
-  color: #396df2;
-  margin-bottom: 32px;
+.seventh-sheet__button:hover {
+  background-color: green;
+  border: 20px;
 }
```
Comment (m05e5) [inline style.css]: - [ ] Your design looks good but you are required to follow the Figma design. So please make sure to use this design pattern for your buttons
**Figma design**
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @robinbenitezmora , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] _Please have a look at the comments under this review_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (robinbenitezmora) [reply index.html]: After Linters test it says that 'Element "submit' not allowed as a child of element "form" in this context'

File: index.html (hunk 1 above)
Comment (robinbenitezmora) [reply index.html]: This is the result after of linters test. Apparently, the 'URL' is not recognized
Screenshot 2022-04-27 155836

File: index.html [hunk 5]
```diff
@@ -160,19 +160,13 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
...
-        <input class="form__button-area form__button-text" type="email" id="e-mail" placeholder="user_name@mail.com">
-        </label>
-      <label for="message-area">
-          <input class="form__button-area--size form__button-text" type="text" id="message-area" placeholder="Write your message here">
-      </label>
+    <form action="https://formspree.io/f/" class="form" method="post">
+      <input class="form__field-area form__field-text" name="name" type="text" id="name" maxlength="30" placeholder="Your name" required>
+      <input class="form__field-area form__field-text" name="email" type="email" id="e-mail" placeholder="user_name@mail.com" required>
+      <textarea class="form__field-area-message" name="area" id="area" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
       <a href="#" class="seventh-sheet__button seventh-sheet__button-text">Get It Touch</a>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please use `button` tag or `input` tag instead of `a` tag and give it type="submit".

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @robinbenitezmora  and @kiboma2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @robinbenitezmora, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**